### DISCUSSION TITLE
Accessibility: disorderly buttons

### DISCUSSION THREAD

**Discussion by grahamperrin** [community/community#5344] [Accessibility]
From <https://github.com/github/feedback/discussions/5249#discussioncomment-1249045>: 

> 2. keying <kbd>Tab</kbd> _once_ to reach the _second_ invisible button

This reveals an accessibility issue: 

* the effect of tabbing is disorderly. 

Reproducible without the invisible button bug: 

1. edit a discussion
2. click the white space to the left of the two buttons
3. <kbd>Tab</kbd> – should reach the first button, wrongly reaches the second
4. <kbd>Tab</kbd> – should reach the second button, wrongly reaches the first. 

![image](https://user-images.githubusercontent.com/192271/131240790-8e11df34-6a51-4fb8-93d1-ed8c53bbac7e.png)

**Comment by cehfisher** [comment]
Hello @grahamperrin - Thank you for your feedback! We couldn't replicate the reported bug. Can you still reproduce the issue? If it's resolved, please close the report. Thanks!
